Source/PeleLMeX_Regrid.cpp

```diff
@@ -288,6 +288,16 @@ PeleLM::regrid(int lbase, amrex::Real time, bool initial)
     }
 
     finest_level = new_finest;
+
+#ifdef PELE_USE_RADIATION
```

Comment:
Now the PeleLMeX example 'tests' do not cover any EB case with soot and radiation modules on for compiling. I am running two production runs (one without EB, one with EB) right now to see whether there is any other issues. I would recommend we have a EB test with soot and radiation modules on first before getting rid of the macro. What do you think?

Reply:
Despite the lack of test coverage for EB with radiation, removing the macro would allow some parts of the CI like the `clang-tidy` check to cover more of the radiation stuff. And if the runtime options default to off, having these compiled in but inactive shouldn't affect any cases being run. It would be great to add EB tests for these capabilities if you have suitable cases though! In any event, this is tangential to the present PR, which appears to be working with the PelePhysics update so I will approve and merge.